docs/faq/README.md

```diff
@@ -1,3 +1,7 @@
+---
+sidebarDepth: 0
```

Comment:
What it is used for exactly ?

Reply:
In markdown each title have a depth.

```
# DEPTH = 1
## DEPTH = 2
```

With that code I specify that I don't want my sidebar display sub menu for title more than 0 because It seems better not to display submenu for that section.

![with_sub_menu](https://user-images.githubusercontent.com/4435536/56847569-dd83b180-68dc-11e9-828e-b07e90b7fe32.png)
![without_sub_menu](https://user-images.githubusercontent.com/4435536/56847571-df4d7500-68dc-11e9-9b84-8ed2bfaed172.png)

All you need to know about that is here : https://vuepress.vuejs.org/default-theme-config/#nested-header-links

Reply:
Cool ! I think you could use depth 2 for the `PWA` part in `Guide` section

Reply:
Currently I did a basic list, which is not the best option ...

Reply:
Yes, you are right. I'll correct that with another PR.